openapi.yml

```diff
@@ -363,7 +363,23 @@ components:
       type: string
       pattern: '^245[0-9]{8}$'
       example: '24503259768'
-    CatalogLink:
+    CreatedInFolioIdentifier:
```

Comment:
Can these be mixed in alphabetically like rest of classes?

Reply:
The current components aren't super consistent about this but I can toss a commit at this branch that (re-)alphabetizes the components, once I get feedback on the new approach to custom types and union types.

Reply:
it would be good to check with Darsi on whether it's going to be "in" and stay that way in "perpetuity" with Folio